qp/drivers/density/qubit.py

```diff
@@ -33,12 +36,12 @@ def __pow__(self, n):
         if n == 1:
             return self
         elif n > 1:
-            return self * (self ** (n-1))
+            return self * (self ** (n - 1))
 
         raise AssertionError("Qubits ** %d not supported" % n)
 
     def __str__(self):
-        return 'Qubits(%d)' % self.n
+        return "Qubits(%d)" % self.n
```

Comment:
Is this from PEP?

Reply:
I think it's from black autoformatting, not sure if it's required by PEP or not